DISPLAY "(null)" invalid; disabling X11 forwarding
I tensorflow/stream_executor/dso_loader.cc:108] successfully opened CUDA library libcublas.so locally
I tensorflow/stream_executor/dso_loader.cc:108] successfully opened CUDA library libcudnn.so locally
I tensorflow/stream_executor/dso_loader.cc:108] successfully opened CUDA library libcufft.so locally
I tensorflow/stream_executor/dso_loader.cc:108] successfully opened CUDA library libcuda.so locally
I tensorflow/stream_executor/dso_loader.cc:108] successfully opened CUDA library libcurand.so locally
INFO:tensorflow:PS hosts are: ['raiders1:2200']
INFO:tensorflow:Worker hosts are: ['raiders1:2201', 'raiders1:2202', 'raiders1:2203', 'raiders1:2204', 'raiders2:2205', 'raiders2:2206', 'raiders2:2207', 'raiders2:2208', 'raiders3:2209', 'raiders3:2210', 'raiders3:2211', 'raiders3:2212', 'raiders8:2213', 'raiders8:2214', 'raiders8:2215', 'raiders8:2216']
E tensorflow/stream_executor/cuda/cuda_driver.cc:491] failed call to cuInit: CUDA_ERROR_NO_DEVICE
I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:153] retrieving CUDA diagnostic information for host: raiders3
I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:160] hostname: raiders3
I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:185] libcuda reported version is: Not found: was unable to find libcuda.so DSO loaded into this program
I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:347] driver version file contents: """NVRM version: NVIDIA UNIX x86_64 Kernel Module  367.27  Thu Jun  9 18:53:27 PDT 2016
GCC version:  gcc version 4.8.4 (Ubuntu 4.8.4-2ubuntu1~14.04.3) 
"""
I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:189] kernel reported version is: 367.27.0
I tensorflow/core/common_runtime/gpu/gpu_init.cc:81] No GPU devices available on machine.
I tensorflow/core/distributed_runtime/rpc/grpc_channel.cc:206] Initialize HostPortsGrpcChannelCache for job ps -> {raiders1:2200}
I tensorflow/core/distributed_runtime/rpc/grpc_channel.cc:206] Initialize HostPortsGrpcChannelCache for job worker -> {raiders1:2201, raiders1:2202, raiders1:2203, raiders1:2204, raiders2:2205, raiders2:2206, raiders2:2207, raiders2:2208, raiders3:2209, raiders3:2210, raiders3:2211, localhost:2212, raiders8:2213, raiders8:2214, raiders8:2215, raiders8:2216}
I tensorflow/core/distributed_runtime/rpc/grpc_server_lib.cc:202] Started server with target: grpc://localhost:2212
INFO:tensorflow:Learning rate: 0.001000, momentum: 0.600000
INFO:tensorflow:Sync mode!!!!!!
INFO:tensorflow:Compute groups : 2
INFO:tensorflow:SyncReplicas enabled: replicas_to_aggregate=16; total_num_replicas=16
INFO:tensorflow:Num nodes per compute groups: 8
INFO:tensorflow:Tokens per step: 8
INFO:tensorflow:2016-07-27 03:15:27.617847 Supervisor
E0727 03:15:43.982443141    7852 tcp_client_posix.c:191]     failed to connect to 'ipv4:172.24.75.41:2205': timeout occurred
E0727 03:15:43.982563433    7852 tcp_client_posix.c:191]     failed to connect to 'ipv4:172.24.75.41:2206': timeout occurred
E0727 03:15:43.982581180    7852 tcp_client_posix.c:191]     failed to connect to 'ipv4:172.24.75.41:2207': timeout occurred
E0727 03:15:43.982595707    7852 tcp_client_posix.c:191]     failed to connect to 'ipv4:172.24.75.41:2208': timeout occurred
E0727 03:15:44.981182495    7852 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2205': socket error: 113
E0727 03:15:44.981242104    7852 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2206': socket error: 113
E0727 03:15:44.981257016    7852 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2207': socket error: 113
E0727 03:15:44.981293096    7852 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2208': socket error: 113
E0727 03:15:48.853120193    7852 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2208': socket error: 113
E0727 03:15:48.853155913    7852 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2205': socket error: 113
E0727 03:15:48.853167572    7852 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2206': socket error: 113
E0727 03:15:48.853177250    7852 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2207': socket error: 113
E0727 03:15:54.853094035    7852 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2205': socket error: 113
E0727 03:15:54.853129412    7852 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2208': socket error: 113
E0727 03:15:54.853158944    7852 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2206': socket error: 113
E0727 03:15:54.853174106    7852 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2207': socket error: 113
E0727 03:16:03.989126951    7852 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2208': socket error: 113
E0727 03:16:03.989189242    7852 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2205': socket error: 113
E0727 03:16:06.989034597    7852 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2206': socket error: 113
E0727 03:16:10.001022978    7852 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2207': socket error: 113
E0727 03:16:16.501076972    7852 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2208': socket error: 113
E0727 03:16:16.501119898    7852 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2205': socket error: 113
E0727 03:16:26.257022402    7852 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2206': socket error: 113
E0727 03:16:26.257067041    7852 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2207': socket error: 113
E0727 03:16:35.257111452    7852 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2208': socket error: 113
E0727 03:16:42.009031490    7852 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2205': socket error: 113
E0727 03:16:53.713040984    7852 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2206': socket error: 113
E0727 03:17:00.573024771    7852 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2207': socket error: 113
run-expr-32node.sh: line 23:  6803 Terminated              CUDA_VISIBLE_DEVICES='' bazel-bin/inception/imagenet_distributed_train --batch_size=16 --data_dir=/lfs/local/0/daniter/16node-data-$2 --job_name='worker' --task_id=$2 --ps_hosts='raiders1:2200' --worker_hosts='raiders1:2201,raiders1:2202,raiders1:2203,raiders1:2204,raiders2:2205,raiders2:2206,raiders2:2207,raiders2:2208,raiders3:2209,raiders3:2210,raiders3:2211,raiders3:2212,raiders8:2213,raiders8:2214,raiders8:2215,raiders8:2216' --initial_learning_rate=$3 --momentum=$4 --sync=True --compute_groups=$5
